src/main/java/com/microsoft/azure/spring/data/cosmosdb/repository/support/DocumentDbEntityInformation.java

```diff
@@ -155,7 +151,8 @@ private String getCollectionName(Class<?> domainClass) {
     private Field getPartitionKeyField(Class<?> domainClass) {
         Field keyField = null;
 
-        final List<Field> fields = FieldUtils.getFieldsListWithAnnotation(domainClass, PartitionKey.class);
+        final List<Field> fields = FieldUtils.getFieldsListWithAnnotation(domainClass,
+                com.microsoft.azure.spring.data.cosmosdb.core.mapping.PartitionKey.class);
```

Comment:
`com.microsoft.azure.spring.data.cosmosdb.core.mapping` package name is still required?